packages/container/lib/registry.js

```diff
@@ -24,10 +24,6 @@ export default class Registry {
     this.fallback = options.fallback || null;
     this.resolver = options.resolver || null;
 
-    if (typeof this.resolver === 'function') {
```

Comment:
I think we are going to have to put an ENV guard around this. [Here is an example of how](https://github.com/thoov/ember.js/blob/64d4708273d86137f8d2e8e28b3303316b3270e4/packages/ember-debug/lib/warn.js#L90) I just did a similar thing.

Reply:
Also here is an example of adding the config to [ember-2-legacy](https://github.com/emberjs/ember-2-legacy/pull/9)

Reply:
@thoov @rwjblue I have updated the PR with the ENV guard, Please have a look.